README.md

```diff
@@ -1,106 +1,91 @@
 # Terraform provider for DataRobot (Preview)
 
-`terraform-provider-datarobot` is the [Terraform provider](https://developer.hashicorp.com/terraform/language/providers) for the [DataRobot API](https://docs.datarobot.com/en/docs/api/index.html).
+`terraform-provider-datarobot` is a [Terraform provider](https://developer.hashicorp.com/terraform/language/providers) for interacting with the [DataRobot API](https://docs.datarobot.com/en/docs/api/index.html).
```

Comment:
I liked it as it was before. There isn't another DR TF provider, and it isn't really "interacting with" as much as it is the TF provider for DataRobot. In fact, maybe this is better:

```suggestion
`terraform-provider-datarobot` is the [Terraform provider](https://developer.hashicorp.com/terraform/language/providers) for [DataRobot](https://docs.datarobot.com/en/docs/api/index.html).
```